.github/workflows/format.yml

```
@@ -14,23 +14,22 @@ jobs:
     name: Format (Python)
     runs-on: ubuntu-22.04
     steps:
-      - name: Set up Python
-        uses: actions/setup-python@v4
-        with:
-          python-version: '3.11'
 
-      - name: Install dependencies
-        run:
-            python -m pip install click==8.0.4 black==23.7.0 isort==5.13.2
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```
Let's drop usage of `checkout@v3` across all our workflows as it uses a deprecated version of NodeJS